docs/redoc/master/openapi.json

```diff
@@ -161,6 +161,13 @@
           "distance": {
             "$ref": "#/components/schemas/Distance"
           },
+          "shard_number": {
+            "default": 1,
+            "description": "Number of shards the collection has",
+            "format": "uint32",
+            "minimum": 0,
```

Comment:
Do we validate that it can't be 0?

Reply:
User cannot set it for now, it is only possible to set it in tests. Actually I did not plan to change the API in this PR, so left it for the next.

Reply:
I guess I can introduce [NonZeroU32](https://doc.rust-lang.org/std/num/struct.NonZeroU32.html) in the next PRs